### ISSUE TITLE
Add Redpanda Console to helm chart

### ISSUE THREAD

**Issue by vuldin** [redpanda-data/helm-charts#112]
## Acceptance criteria

Helm chart definition need to be moved from https://github.com/redpanda-data/console/tree/master/helm
to this repo. 

The automated process to release console helm chart to Redpanda Data helm index.yaml should be created.

## More context

Add as a new section in `values.yaml`':
```yaml
listeners:
  console:
    enabled: true
    kafkaEndpoint: default
    port: 8080
    external:
      enabled: true
    tls:
      cert: console
      requireClientAuth: false
```

- verify `requireClientAuth` makes sense in the context of Console
- look into configuring the `redpanda` service so that kafka/admin/rpc/etc ports point to `redpanda-0` (and other brokers), and have port 8080 of same service point to the console pod
- verify console only supports running on a single port

**Comment by vuldin** [comment]
Details on running console: https://github.com/redpanda-data/console

**Comment by rkruze** [comment]
Would we want this to be just a listener or something that gets installed on a single node?

**Comment by joejulian** [comment]
fixed in #328
